stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -14,6 +14,13 @@ class AsyncResult():
     def ready(self) -> bool: ...
     def successful(self) -> bool: ...
 
+class Event():
+    def __init__(self, *, ctx: Any) -> None: ...
```

Comment:
I think it's supposed to be a `BaseContext`, but we may not have that type in the stubs.